### PR TITLE
DB schemas and configs on git integration

### PR DESCRIPTION
**Pull request by Thumimku**
### Proposed changes in this pull request
This PR is add needed db schemas and configs on [identity-fetch-remote](https://github.com/wso2/product-is/issues/8052) feature.

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1532,6 +1532,56 @@ ALTER TABLE IDN_CONFIG_FILE ADD CONSTRAINT RESOURCE_ID_FILE_FOREIGN_CONSTRAINT F
 IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 /
 
+CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
```
Comment (nilasini) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1535]: Formatting

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql [hunk 2]
```diff
@@ -1544,6 +1544,55 @@ CREATE TABLE IDN_CONFIG_FILE (
 )
 /
 
+CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
```
Comment (nilasini) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql:1547]: Formatting check other table creation as well

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql (hunk 1 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1535]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql (hunk 2 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql:1547]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

**Pull request merged**